distributed/stealing.py

```diff
@@ -338,7 +338,7 @@ async def move_task_confirm(self, *, key, state, stimulus_id, worker=None):
                 self.log(("confirm", *_log_msg))
```

Comment:
There are a hand full of places where we update the occupancy. I recently factored some of this out but the remaining places are hard to unify and I doubt it will be worth the trouble.

Is this what you mean or something else?